### DISCUSSION TITLE
Is there any plan to support Isaacsim 2023.1.1?

### DISCUSSION THREAD

**Discussion by yizhengzhang1** [NVlabs/curobo#173] [Software Q&A] [answered]
Hi, 
In the install document of curobo: https://curobo.org/source/getting_started/1_install_instructions.html,
it said 

> Isaac Sim version: cuRobo is stable in Isaac Sim 2022.2.1. While all examples will work in 2023.1.0, you might have some instability as it has not been heavily tested. The examples will not work in 2023.1.1.

Will it be support by 2023.1.1 later?

**Comment by eferreirafilho** [comment]
I'm wondering the same. Particularly, I want to be able to use 2023.1.1 DeformableView

**Comment by balakumar-s** [comment, marked as answer]
We added partial support for 2023.1.1. We haven't gotten Franka Panda robot to work well but other robots work now.
